crates/fmt/src/formatter.rs

```diff
@@ -2093,15 +2093,11 @@ impl<W: Write> Visitor for Formatter<'_, W> {
         let (ident, string) = (ident.safe_unwrap(), string.safe_unwrap());
         return_source_if_disabled!(self, loc, ';');
 
-        let pragma_descriptor = if ident.name == "solidity" {
-            // There are some issues with parsing Solidity's versions with crates like `semver`:
-            // 1. Ranges like `>=0.4.21<0.6.0` or `>=0.4.21 <0.6.0` are not parseable at all.
-            // 2. Versions like `0.8.10` got transformed into `^0.8.10` which is not the same.
-            // TODO: semver-solidity crate :D
-            &string.string
-        } else {
-            &string.string
-        };
+        // There are some issues with parsing Solidity's versions with crates like `semver`:
+        // 1. Ranges like `>=0.4.21<0.6.0` or `>=0.4.21 <0.6.0` are not parseable at all.
+        // 2. Versions like `0.8.10` got transformed into `^0.8.10` which is not the same.
+        // TODO: semver-solidity crate :D
+        let pragma_descriptor = &string.string;
```

Comment:
We can remove the comment here imo as it is not relevant to conditional logic anymore